libr/util/file.c

```diff
@@ -1052,8 +1059,8 @@ R_API bool r_file_copy (const char *src, const char *dst) {
 #if HAVE_COPYFILE_H
 	return copyfile (src, dst, 0, COPYFILE_DATA | COPYFILE_XATTR) != -1;
 #elif __WINDOWS__
-	char *s = r_file_abspath (src);
-	char *d = r_file_abspath (dst);
+	PTCHAR s = r_sys_conv_utf8_to_utf16 (src);
```

Comment:
Currently radare2 can be built with UNICODE and without, please add corresponding checks, if it is defined then do conv